lib/apiservers/engine/backends/container/viccontainer.go

```diff
@@ -24,6 +24,7 @@ import (
 type VicContainer struct {
 	Name        string
 	ImageID     string
+	LayerID     string // used to find the vmdk to use with this container
```

Comment:
Might consider adding a comment for the ImageID..just to be clear...and then for LayerID modify the comment to note that it is the child most ID for vmdk identification